packages/fxa-settings/src/pages/Signin/ConfirmSignin/index.tsx

```diff
@@ -8,21 +8,22 @@ import { RouteComponentProps } from '@reach/router';
 import ConfirmWithLink, {
   ConfirmWithLinkPageStrings,
 } from '../../../components/ConfirmWithLink';
+import { REACT_ENTRYPOINT } from '../../../constants';
 
 export type ConfirmSigninProps = {
   email: string;
   goBackCallback?: () => void;
   withWebmailLink?: boolean; // TO-DO: Replace broker functionality which gives us this value (provider?)
 };
 
+export const routeName = 'confirm-signin';
```

Comment:
Q: `routeName` or `viewName`? I think this was one of 3 references to `const routeName` and the other 2 were changed. 👀

Reply:
Ah yeah, I missed this. This was discussed a little in a Slack group chat, we already have a lot of these called `viewName` but I'd called a couple `routeName` before realizing we're supposed to go kebab-case here. We could also go with `pageName` potentially but I think `viewName` is preferred by the majority.